src/Compilers/CSharp/Portable/CSharpResources.resx

```diff
@@ -1221,7 +1221,7 @@
   <data name="ERR_AttributeCantBeGeneric" xml:space="preserve">
     <value>Cannot apply attribute class '{0}' because it is generic</value>
   </data>
-  <data name="ERR_DuplicateBound" xml:space="preserve">
+  <data name="ERR_DuplicateBound" xml:space="preserve">                                
```

Comment:
>  <data name="ERR_DuplicateBound" xml:space="preserve">          [](start = 0, length = 64)

It looks like there are no meaningful changes in this file. Consider reverting

Reply:
Will do